docs/psmdb-pro.md

```diff
@@ -15,6 +15,7 @@ Find the list of solutions available in Percona Server for MongoDB Pro builds:
 | Name                                | Version added | Description  | 
 | ----------------------------------- | ------------- | -------------
 | [FIPS support ](fips.md)| [7.0.4-2](release_notes/7.0.4-2.md) | FIPS mode provides a way to use FIPS-compliant encryption and run the Percona Server for MongoDB with the FIPS-140 certified library for OpenSSL. This helps customers meet minimum security requirements for cryptographic modules and testing in both hardware and software. |
+| [File copy based initial sync](initial-sync.md) | [7.0.21-12](release_notes/7.0.21-12.md) | File copy based initial sync is a new method of syncing data between nodes in a cluster. It is faster than the logical sync method as it copies physical files rather than cloning data. |
```

Comment:
Consider replacing "a new method" with "an additional method"

Reply:
"method of syncing data between nodes in a cluster" might be misleading. More technically correct way to say this is:
> method of doing initial sync for the new node in a replica set